DTML.EduBot/Dialogs/RootDialog.cs

```diff
@@ -218,7 +143,7 @@ public Task StartAsync(IDialogContext context)
 
             try
             {
-                using (var scope = WebApiApplication.FindContainer().BeginLifetimeScope())
+                using(var scope = WebApiApplication.FindContainer().BeginLifetimeScope())
```

Comment:
If we are introducing DI via constructor instead of service locator pattern through scope runtime resolution, why not DI via constructor consistently?